buildlib/azure-pipelines-perf.yml

```diff
@@ -128,5 +130,13 @@ stages:
             clean: true
           - bash: |
               set -x
+              echo 'Retain 3 latest workdirs for manual repro; clean the rest'
+              wrk_dirs="/hpc/scrap/azure/$(Build.DefinitionName)/"
+              for dir in $(ls -lt "$wrk_dirs" | tail -n +4 | awk '{print $NF}'); do
+                  echo "Removing old workdir: ${wrk_dirs}${dir}"
+                  rm -rf "${wrk_dirs}${dir}"
+              done
+
+
```

Comment:
should we remove line 141?

Reply:
OMG, it literally defeats the PR's purpose!
Thanks a lot for caching it.